### PR TITLE
refactor: broadcast transaction after adding final signature

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
https://app.clickup.com/t/vhp6hr

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by samharperpittam** [comment]
@goga-m - Can you try sending a transaction from a Multisig wallet? I don't think it's possible anymore. I've signed a transaction with all participants, the transaction has a "Ready to Broadcast" status, but there's no method of being able to send as far as i can see.

**Comment by goga-m** [comment]
@samharperpittam :hmhm: the `Ready to broadcast` status should not be displayed as the `Awaiting final Signature` is now considered as a final status. 

Seems that the final signature was added but it didn't manage to broadcast. Will have a look and patch a fix to handle this particular case (signed with final but not broadcasted for some reason e.g server error).

**Comment by goga-m** [comment]
@samharperpittam should be fixed now

File: src/domains/transaction/components/TransactionTable/PendingTransactionsTable/__snapshots__/PendingTransactionsTable.test.tsx.snap [hunk 1]
```diff
@@ -866,7 +866,7 @@ exports[`Signed Transaction Table should render signed transactions and handle e
                 class="TableCell__TableCellInnerWrapper-sc-v39gua-0 cDWumh justify-center truncate"
               >
                 <span
-                  class="p-1 text-theme-secondary-700"
+                  class="\\"p-1 text-theme-secondary-700"
```
Comment (alesmit) [inline src/domains/transaction/components/TransactionTable/PendingTransactionsTable/__snapshots__/PendingTransactionsTable.test.tsx.snap:869]: @goga-m any idea on what's causing this double backslash here? 🤔

File: src/domains/transaction/components/TransactionTable/TransactionRow/SignedTransactionRow.tsx [hunk 2]
```diff
@@ -171,7 +124,11 @@ export const SignedTransactionRow = ({
 			)}
 
 			<TableCell className="w-16" innerClassName="justify-center truncate" isCompact={isCompact}>
-				<StatusLabel wallet={wallet} transaction={transaction} />
+				<Tooltip content={status.label}>
+					<span className={`"p-1 ${status.className}`}>
```
Comment (alesmit) [inline src/domains/transaction/components/TransactionTable/TransactionRow/SignedTransactionRow.tsx:128]: @goga-m must be this! 🦅

Comment (goga-m) [reply src/domains/transaction/components/TransactionTable/TransactionRow/SignedTransactionRow.tsx:128]: You got this right @alesmit :100: Thanks!

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**